community/examples/hpc-build-slurm-image.yaml

```diff
@@ -0,0 +1,116 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+blueprint_name: hpc-build-slurm-image
+
+vars:
+  project_id: ns-playground-2023-01-19  ## Set GCP Project ID Here ##
+  deployment_name: build-slurm-1
+  region: us-central1
+  zone: us-central1-a
+
+  image_build_machine_type: n2d-standard-32
+  build_from_image_family: hpc-rocky-linux-8
+  build_from_image_project: cloud-hpc-image-public
+  built_image_family: my-custom-slurm
+  built_instance_image:
+    family: $(vars.built_image_family)
+    project: $(vars.project_id)
+  instance_image_custom: true
+
+deployment_groups:
+- group: setup
+  modules:
+  - id: network
+    source: modules/network/vpc
+
+  - id: slurm-build-script
+    source: modules/scripts/startup-script
+    settings:
+      runners:
```

Comment:
I would set `install_ansible: false` explicitly and add a comment

- to avoid breaking upon nay future changes in behavior of startup-script
- allow user to add an ansible runner without breaking script (I bet it would work if it came after your installation)
- generally be explicit about a choice that matters